package.json

```diff
@@ -30,6 +31,11 @@
     "underscore": "^1.9.1"
   },
   "devDependencies": {
+    "babel-cli": "^6.26.0",
+    "babel-core": "^6.26.3",
+    "babel-loader": "^8.0.5",
```

Comment:
What is the function of the `babel-loader` dependency?

Reply:
Yeah, but we aren't using webpack at this time.

Reply:
okay, i will remove and push changes.

Reply:
I have pushed the change.